src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -105,6 +116,63 @@ def _load_version(cls, state, version):
         # we need to undo what we did at save and turn the proxy object
         # back into a Python class
         state['detector'] = _ObjectDetector._load_version(
-            state['detector'], state["detector_version"])
+            state['detector'], state["_detector_version"])
         return OneShotObjectDetector(state)
 
+    def __str__(self):
+        """
+        Return a string description of the model to the ``print`` method.
+
+        Returns
+        -------
+        out : string
+            A description of the OneShotObjectDetector
+        """
+        return self.__repr__()
+
+    def __repr__(self):
+        """
+        Print a string description of the model when the model name is entered
+        in the terminal.
+        """
+        width = 40
+        sections, section_titles = self._get_summary_struct()
+        detector = self.__proxy__['detector']
+        out = _tkutl._toolkit_repr_print(detector, sections, section_titles,
+                                         width=width, class_name='OneShotObjectDetector')
+        return out
+
+    def _get_summary_struct(self):
+        """
+        Returns a structured description of the model, including (where
+        relevant) the schema of the training data, description of the training
+        data, training statistics, and model hyperparameters.
+
+        Returns
+        -------
+        sections : list (of list of tuples)
+            A list of summary sections.
+              Each section is a list.
+                Each item in a section list is a tuple of the form:
+                  ('<label>','<field>')
+        section_titles: list
+            A list of section titles.
+              The order matches that of the 'sections' object.
+        """
+        model_fields = [
+            ('Number of classes', 'num_classes'),
+            ('Input image shape', 'input_image_shape'),
+        ]
+        data_fields = [
+            ('Number of synthetically generated examples', 'num_examples'),
+            ('Number of synthetically generated bounding boxes', 'num_bounding_boxes'),
```

Comment:
Where are the num_examples and num_bounding_boxes fields getting set?

Reply:
They are from the detector model.

Reply:
Do we need to save the 'num_examples' and 'num_bounding_boxes' fields for the OneShotObjectDetector? Currently, they are fields on the 'detector' field of the OneShotObjectDetector.

Reply:
I haven't added anything to the model. Just showing things in the __repr__